src/components/inputs/TokenSelectInput/TokenSelectInput.vue

```diff
@@ -91,7 +93,7 @@ const toggleModal = () => {
     <teleport to="#modal">
       <SelectTokenModal
         v-if="openTokenModal"
-        :excludedTokens="[modelValue]"
+        :excludedTokens="[...excludedTokens, modelValue]"
```

Comment:
By default `excludedTokens` is defined as an empty array so this would always work.